Content.Shared/GameObjects/Components/ActionBlocking/SharedCuffedComponent.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.Maths;
+using Robust.Shared.ViewVariables;
+using System;
+
+namespace Content.Shared.GameObjects.Components.ActionBlocking
+{
+    public class SharedCuffedComponent : Component, IActionBlocker
```

Comment:
And `CuffedComponent` could definitely use a better name, since it implies that the mob is cuffed.
Like how `StunnableComponent` isn't `StunnedComponent`.